### PR TITLE
Add testcases for AddWorkExperienceCommand and AddWorkExperienceCommandParser

### LINKED ISSUE #88: Add testcase for add work experience command
due:Thursday

### PR DESCRIPTION
**Pull request by kuiktzecheng**
Fixes #88 
due:Thursday

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/AddWorkExperienceCommandTest.java [hunk 1]
```diff
@@ -44,6 +44,9 @@ public void execute_addWorkExperience_successful() throws Exception {
 
         // Verify that the person in the model has been updated
         assertEquals(editedPerson, model.getFilteredPersonList().get(0));
+
+        // Additional check to cover the last assertion
```
Comment (jeremychiaaaaa) [inline src/test/java/seedu/address/logic/commands/AddWorkExperienceCommandTest.java:48]: Great use of assert here!

File: src/test/java/seedu/address/logic/parser/AddWorkExperienceCommandParserTest.java [hunk 2]
```diff
@@ -88,4 +88,11 @@ public void parseInvalidArgs_nonNumericIndex_throwsParseException() {
                 String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddWorkExperienceCommand.MESSAGE_USAGE));
     }
 
+    @Test
+    public void parseInvalidArgs_emptyWorkExperience_throwsParseException() {
+        // Additional case for an empty work experience after the prefix
+        assertParseFailure(parser, " in/1 w/", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                AddWorkExperienceCommand.MESSAGE_USAGE));
+    }
 }
```
Comment (jeremychiaaaaa) [inline src/test/java/seedu/address/logic/parser/AddWorkExperienceCommandParserTest.java:97]: Great job with this test case!

**Review by jeremychiaaaaa — APPROVED** [review]
lgtm!

File: src/test/java/seedu/address/logic/commands/AddWorkExperienceCommandTest.java [hunk 3]
```diff
@@ -44,6 +44,9 @@ public void execute_addWorkExperience_successful() throws Exception {
...
+        assertTrue(model.getFilteredPersonList().contains(editedPerson));
     }
 
     @Test
```
Comment (arnav12344) [inline src/test/java/seedu/address/logic/commands/AddWorkExperienceCommandTest.java:52]: Great job with this test case!!

**Pull request merged**